runners/spark/src/test/java/org/apache/beam/runners/spark/translation/streaming/SimpleStreamingWordCountTest.java

```diff
@@ -34,7 +36,9 @@
 import org.apache.beam.sdk.transforms.windowing.FixedWindows;
 import org.apache.beam.sdk.transforms.windowing.Window;
 import org.apache.beam.sdk.values.PCollection;
+
```

Comment:
To the best of my knowledge, the convention for imports now is no spaces (except maybe to differ from static imports).